src/main/java/com/revature/studyforce/flashcard/dto/VoteDTO.java

```diff
@@ -0,0 +1,43 @@
+package com.revature.studyforce.flashcard.dto;
+
+import com.revature.studyforce.flashcard.model.Vote;
+import lombok.*;
+import org.springframework.util.Assert;
+import java.util.function.Function;
+
+/**
+ * Data Transfer Object to carry the Vote model's information to and from the front end
+ * {@link Vote}
+ * @author Elizabeth Ye
+ */
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@ToString
+public class VoteDTO {
+
+    /**
+     * Fields used in UI layer
+     */
+    private int answerId;
+    private int userId;
+    private int value;
+
+    /**
+     * function to take a Vote model and if not null return it as a data transfer object
+     * @return a new VoteDTO with the corresponding fields
+     */
+    public static Function<Vote,VoteDTO> convertVoteToDto(){
```

Comment:
incorrect documentation, your method does not return a new VoteDTO. The method acts as a getter, returning a function which can then convert a vote to a DTO